core_backend/migrations/versions/50caccd9e332_changed_response_feedback_schema.py

```diff
@@ -0,0 +1,38 @@
+"""Changed Response Feedback Schema
+
+Revision ID: 50caccd9e332
+Revises: f269c75dbf69
+Create Date: 2024-04-19 10:30:54.214343
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "50caccd9e332"
+down_revision: Union[str, None] = "f269c75dbf69"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "feedback", sa.Column("feedback_sentiment", sa.String(), nullable=True)
+    )
```

Comment:
cf: 
<img width="597" alt="image" src="https://github.com/IDinsight/aaq-core/assets/7042047/0e3989d8-2913-4c5e-b19d-301f45c92787">

Reply:
Ah I see if I just don't pass the `feedback_sentiment` key at all then it works (picks it up as Unknown). I like the flexibility of collecting just up/down and/or text feedback.. 
Can we add to API dos that if you want to just provide text feedback you need to not pass "feedback_sentiment" (and not set it as null or ""), OR change the schema so that null -> Unknonw or something?

Reply:
Added to API docs